### ISSUE TITLE
Crashed: com.apple.StoreKit.SKAdNetworkImpressionQueue

### ISSUE THREAD

**Issue by aplusm** [inner-active/InneractiveAdSDK-iOS#36]
Hi,

With Fyber_Marketplace_SDK V8.1.2, we are seeing a huge number of crash with the following stack trace:

```
Crashed: com.apple.StoreKit.SKAdNetworkImpressionQueue
0  libobjc.A.dylib                0x5568 objc_opt_respondsToSelector + 48
1  Foundation                     0x1b894 _NSDescriptionWithStringProxyFunc + 60
2  CoreFoundation                 0x99f4 __CFStringAppendFormatCore + 8144
3  CoreFoundation                 0x290e0 _CFStringCreateWithFormatAndArgumentsReturningMetadata + 172
4  <OUR_APP>                      0xd1f514 -[FMPLogger logMessageWithLevel:text:] + 3337964
5  <OUR_APP>                      0xd0d168 __49-[IAContentController startStorekitAdImpression:]_block_invoke + 3263296
6  StoreKit                       0x28cfc __49+[SKAdNetwork startImpression:completionHandler:]_block_invoke_2 + 196
7  CoreFoundation                 0x223a4 __invoking___ + 148
8  CoreFoundation                 0x3fb74 -[NSInvocation invoke] + 468
9  Foundation                     0x34f90 __NSXPCCONNECTION_IS_CALLING_OUT_TO_REPLY_BLOCK__ + 24
10 Foundation                     0x55f44 -[NSXPCConnection _decodeAndInvokeReplyBlockWithEvent:sequence:replyInfo:] + 572
11 Foundation                     0x30b14 __88-[NSXPCConnection _sendInvocation:orArguments:count:methodSignature:selector:withProxy:]_block_invoke_3 + 212
12 Foundation                     0x45140 -[NSXPCConnection _sendInvocation:orArguments:count:methodSignature:selector:withProxy:] + 2632
13 Foundation                     0x7e1e4 -[NSXPCConnection _sendSelector:withProxy:arg1:arg2:] + 144
14 Foundation                     0x92c60 _NSXPCDistantObjectSimpleMessageSend2 + 76
15 StoreKit                       0x28c20 __49+[SKAdNetwork startImpression:completionHandler:]_block_invoke.42 + 132
16 libdispatch.dylib              0x2924 _dispatch_call_block_and_release + 32
17 libdispatch.dylib              0x4670 _dispatch_client_callout + 20
18 libdispatch.dylib              0xbdf4 _dispatch_lane_serial_drain + 672
19 libdispatch.dylib              0xc968 _dispatch_lane_invoke + 392
20 libdispatch.dylib              0x171b8 _dispatch_workloop_worker_thread + 656
21 libsystem_pthread.dylib        0x10f4 _pthread_wqthread + 288
22 libsystem_pthread.dylib        0xe94 start_wqthread + 8
```

**Comment by 2energycell** [comment]
@aplusm thanks for the reported issue; is there a way to reproduce the issue? can you please share more details? thanks

**Comment by aplusm** [comment]
Hi @2energycell ,

We don't have any more information to share (beyond this stackTrace from Crashlytics): 

We experienced a lot of crashes due to this issue (more than 18K) so we disabled all Fyber calls in our adUnit (AppLovin' MAX) to protect our users.

**Comment by 2energycell** [comment]
@aplusm what is your app? also please reach the appropriate account manager, the issue is under investigation; thanks;

**Comment by 2energycell** [comment]
the issue is resolved in 8.1.3;
